CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 ## [Unreleased](https://github.com/cucumber/cucumber-ruby/compare/v8.0.0.rc.1...main)
 
 ### Added
+- Add support for TrufflrRuby ([PR#1612](https://github.com/cucumber/cucumber-ruby/pull/1612) [gogainda](https://github.com/gogainda))
```

Comment:
TruffleRuby* and I think the entry should be similar to the ones below (see line 39 for example), so empty line before, PR/author link on a separate line, etc